includes/Checker/Abstract_Check_Runner.php

```diff
@@ -376,6 +376,7 @@ private function register_checks() {
 				'late_escaping'              => new Checks\Late_Escaping_Check(),
 				'plugin_updater'             => new Checks\Plugin_Updater_Check(),
 				'plugin_review_phpcs'        => new Checks\Plugin_Review_PHPCS_Check(),
+				'performant_wp_query_params' => new Checks\Late_Escaping_Check(),
```

Comment:
```suggestion
				'performant_wp_query_params' => new Checks\Performant_WP_Query_Params_Check(),
```
Wrong class assign in https://github.com/10up/plugin-check/pull/193/commits/eeb27193ab1b3d130bfb659e9ffc78c5ed0afc9e